recipes/drmaa/run_test.py

```diff
@@ -0,0 +1,4 @@
+try:
+    import drmaa
```

Comment:
If we really want to test it, we could install libdrmaa and sge with `yum` in the docker image. However, that takes awhile.

I don't know that we want to include that in the docker image by default. Though I do have a docker image for CentOS 6.6, which we could use here. It is setup a bit differently than the one used here and am not sure how much I want to change it for this purpose. However, we could construct a variant of that image for this purpose.

Thoughts?